cmd/zrok/copy.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"fmt"
+	"github.com/openziti/zrok/drives/sync"
+	"github.com/openziti/zrok/environment"
+	"github.com/openziti/zrok/sdk/golang/sdk"
+	"github.com/openziti/zrok/tui"
+	"github.com/spf13/cobra"
+	"net/url"
+	"os"
+)
+
+func init() {
+	rootCmd.AddCommand(newCopyCommand().cmd)
+}
+
+type copyCommand struct {
+	cmd       *cobra.Command
+	sync      bool
+	basicAuth string
+}
+
+func newCopyCommand() *copyCommand {
+	cmd := &cobra.Command{
+		Use:     "copy <source> [<target>] (<target> defaults to 'file://.`)",
+		Short:   "Copy (unidirectional sync) zrok drive contents from <source> to <target> ('http://', 'file://', and 'zrok://' supported)",
```

Comment:
It's unclear that the expected value for the `zrok://` scheme is a public or private access token, or whether any filesystem path within that share can be specified in a `copy` command.

Reply:
Did you look at the docs? I feel like all of these topics are explained there:

https://docs.zrok.io/docs/guides/drives/cli/

Reply:
I did not. I expected the built-in usage hints to be sufficient for understanding the grammar and syntax.

Reply:
There's an office hours and documentation for this feature _specifically_ because it's going to be very difficult to condense this all down into something that you can understand just from `--help`, in the first go.